docs/select-pattern.md

```diff
@@ -21,6 +21,7 @@ through the store (similar to `immutableJS`'s `getIn`).
 - If a `function` is passed the `@select` decorator will attempt to use that function
 as a selector on the RxJs observable.
 - If nothing is passed then the `@select` decorator will attempt to use the name of the class property to find a matching value in the Redux store. Note that a utility is in place here where any $ characters will be ignored from the class property's name.
+- When using with NgIf, use parenthesis to to wrap the observable result before negating the value, e.g. `*ngIf="!(isAuthenticated$ | async)"`.
```

Comment:
This comes up often enough that it might be worth having a short page in the 'in-depth usage' section on patterns for `| async`.

Perhaps linking to the angular.io docs on async pipe, but also showing a couple of examples like this boolean expression or the elvis operator.